Tests/Kitura/TestResponse.swift

```diff
@@ -150,11 +150,41 @@ class TestResponse : XCTestCase {
                     "-----------------------------9051914041544843365972754266\r\n" +
                     "Content-Disposition: form-data; name=\"file1\"; filename=\"a.txt\"\r\n" +
                     "Content-Type: text/plain\r\n\r\n" +
-                    "Content of a.txt.\r\n\r\n" +
+                    "Content of a.txt.\r\n" +
```

Comment:
Yes, multiple empty lines are supported, but then they would end up in the data reflected back by the test server.  So if the \r\n were added back to "Content of a.txt\r\n", it would then appear in the response:

`XCTAssertEqual(body!, "text text(\"text default\") file1 text(\"Content of a.txt.\r\n\") file2 text(\"<!DOCTYPE html><title>Content of a.html.</title>\") ")
`

I removed it from the POST to keep the test case the same.